turbo/src/main/kotlin/dev/hotwire/turbo/nav/TurboNavDestination.kt

```diff
@@ -127,7 +128,12 @@ interface TurboNavDestination {
      * Turbo navigation flow).
      */
     fun shouldNavigateTo(newLocation: String): Boolean {
-        return true
+        return try {
+            URL(newLocation)
```

Comment:
I copied the implementation from https://github.com/hotwired/turbo-android/blob/main/turbo/src/main/kotlin/dev/hotwire/turbo/config/TurboPathConfiguration.kt#L93

Should this use `Uri.parse(newLocation)` instead? I noticed the project uses both.